engine/templates/layout.go.html

```diff
@@ -20,19 +20,25 @@
     <script type="text/javascript" src="vectortile-view-component/main.js"></script>
     <script type="text/javascript" src="vectortile-view-component/polyfills.js"></script>
     <script type="text/javascript" src="vectortile-view-component/runtime.js"></script>
+    <script type="text/javascript">
+        function setLanguage(lang) {
+            document.cookie = 'lang='+lang+';path=/;max-age=${60*60*24};same-site=strict;secure';
```

Comment:
Zou mooi zijn om de max-age uit een constant te halen, en die te delen met de SetCookie in de Go code.